plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcPageSinkProvider.java

```diff
@@ -38,12 +39,24 @@ public JdbcPageSinkProvider(JdbcClient jdbcClient)
     @Override
     public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorOutputTableHandle tableHandle)
     {
-        return new JdbcPageSink(session, (JdbcOutputTableHandle) tableHandle, jdbcClient);
+        throw new IllegalArgumentException("createPageSink not supported without ConnectorPageSinkId");
```

Comment:
please file an issue to remove deprecated methods from the SPI and maybe add a comment:
```
// TODO(issue-number); to be removed
```
by relevant methods in the SPI